capacitor.config.ts

```diff
@@ -2,11 +2,15 @@ import { CapacitorConfig } from '@capacitor/cli'
 
 const config: CapacitorConfig = {
   appId: 'org.phidatalab.radar_armt',
-  appName: 'RADAR Questionnaire',
+  appName: 'RADAR Active RMT',
   webDir: 'www',
   server: {
     androidScheme: 'https'
   },
+  loggingBehavior: 'none',
+  ios: {
+    scheme: 'RADAR Questionnaire'
```

Comment:
I've just removed this since I added the URL schemes directly to the Android and iOS configs. https://github.com/RADAR-base/RADAR-Questionnaire/blob/d55251b9b3256b202edf6c778bae48fa22ea317b/ios/App/App/Info.plist#L30 and https://github.com/RADAR-base/RADAR-Questionnaire/blob/d55251b9b3256b202edf6c778bae48fa22ea317b/android/app/src/main/res/values/strings.xml#L6